test/cpp/run_tests.sh

```diff
@@ -6,6 +6,7 @@ BUILDTYPE="Release"
 VERB=
 FILTER=
 BUILD_ONLY=0
+USE_COVERAGE=1
```

Comment:
Can we set it to 0 as default, and pass `USE_COVERAGE=1` when calling `run_tests.sh` else where? Like, https://github.com/pytorch/xla/blob/1217d33ba9253755b0320fbc71def094d7db5fc6/.circleci/config.yml#L191, guess this may be already doing that?

Reply:
The above goes through. https://github.com/pytorch/xla/blob/1217d33ba9253755b0320fbc71def094d7db5fc6/.circleci/test.sh#L28,

Reply:
Good catch! Removed